cppcache/src/CacheImpl.cpp

```diff
@@ -744,4 +742,21 @@ CacheImpl::getMemberListForVersionStamp() {
   return *versionStampMemIdList;
 }
 
+std::unique_ptr<DataInput> CacheImpl::createDataInput(const uint8_t* buffer,
+                                                      size_t len) const {
+  auto poolName = getPoolManager().getDefaultPool()->getName();
+  return std::unique_ptr<DataInput>(new DataInput(buffer, len, this, poolName));
+}
+
+std::unique_ptr<DataOutput> CacheImpl::createDataOutput() const {
```

Comment:
@pivotal-jbarrett here is the other use of the second constructor

Reply:
which is used quite a bit

Reply:
I meant the default constructor. You showed me the constructor that takes cache and pool.